app/Http/Controllers/API/ChallengeAPIController.php

```diff
@@ -52,4 +50,28 @@ public function viewCurrentYearAttendedEvents(Request $request)
         $events = $this->challengeService->viewCurrentYearAttendedEvents($request->get('volunteer'));
         return response()->json($events);
     }
+
+    private function getResponse($validator){
```

Comment:
Actually you will need to use this function for most APIControllers. You can make an API service and put this function there.